packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.types.ts

```diff
@@ -44,69 +47,87 @@ export interface ITooltipHostProps extends React.HTMLAttributes<HTMLDivElement |
   componentRef?: IRefObject<ITooltipHost>;
 
   /**
-   * Additional properties to pass through for Callout, reference detail properties in ICalloutProps
+   * Additional properties to pass through for Callout.
    */
   calloutProps?: ICalloutProps;
 
   /**
-   * Optionally a number of milliseconds to delay closing the tooltip, so that
-   * the user has time to hover over the tooltip and interact with it. Hovering
-   * over the tooltip will count as hovering over the host, so that the tooltip
-   * will stay open if the user is actively interacting with it.
+   * Number of milliseconds to delay closing the tooltip, so that the user has time to hover over
+   * the tooltip and interact with it. Hovering over the tooltip will count as hovering over the
+   * host, so that the tooltip will stay open if the user is actively interacting with it.
    */
   closeDelay?: number;
 
   /**
-   *  Content to be passed to the tooltip
+   * Content to display in the Tooltip.
    */
   content?: string | JSX.Element | JSX.Element[];
 
   /**
-   * Length of delay
-   * @defaultvalue medium
+   * Length of delay before showing the tooltip on hover.
+   * @defaultvalue TooltipDelay.medium
    */
   delay?: TooltipDelay;
 
   /**
-   * Indicator of how the tooltip should be anchored to its targetElement.
+   * How the tooltip should be anchored to its `targetElement`.
+   * @defaultvalue DirectionalHint.topCenter
    */
   directionalHint?: DirectionalHint;
 
   /**
    * How the element should be positioned in RTL layouts.
-   * If not specified, a mirror of `directionalHint` will be used instead
+   * If not specified, a mirror of `directionalHint` will be used.
    */
   directionalHintForRTL?: DirectionalHint;
 
   /**
-   * Optional class name to apply to tooltip host.
+   * Class name to apply to tooltip host.
    */
   hostClassName?: string;
 
   /**
-   * Only show if there is overflow. If set, the tooltip hosts observes  and only shows the tooltip if this element has overflow.
-   * It also uses the parent as target element for the tooltip.
+   * Class name to apply to the *tooltip itself*, not the host.
+   * To apply a class to the host, use `hostClassName` or `styles.root`.
+   */
+  className?: string;
+
+  /**
+   * If this is unset (the default), the tooltip is always shown even if there's no overflow.
```

Comment:
These strings get placed into the 'description' of the prop. I'm worried as we try to add more 'details', the description is lost, and our props list gets harder to parse. 

![image](https://user-images.githubusercontent.com/1434956/70276067-4d57f580-1764-11ea-8736-bac496c0a34d.png)

Reply:
i agree, this information is useful, but curious if there's a better way to do this in the future.

Reply:
I think the "right" solution here is to introduce line break support in the descriptions...

Reply:
Opened #11388 to track adding line breaks